test/fixtures/assert/assert_throws_stack.snapshot

```diff
@@ -19,16 +19,16 @@ AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
   generatedMessage: true,
   code: 'ERR_ASSERTION',
   actual: Error: foo
-      at assert.throws.bar (*assert_throws_stack.js:*)
-      at getActual (node:assert:*)
-      at Function.throws (node:assert:*)
-      at Object.<anonymous> (*assert_throws_stack.js:*:*)
```

Comment:
Isn't the test checking exactly for this stack?

Reply:
the test should check for it exactly ?, current test generates:
```
  at * {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: Error: foo
      at *
      at *
      at *
      at *
      at *
      at *
      at *
      at *
      at *
      at node:internal*main*run_main_module**,
  expected: { bar: true },
  operator: 'throws'
}
```

Reply:
this is the original PR adding this test: https://github.com/nodejs/node/pull/27243
@BridgeAR is this test intended to snapshot the entire stack trace? is this change ok?

Reply:
Any idea or suggestions to close this PR ?

Reply:
maybe add another trasnform function to make this snapshot similar to the original one?